src/controllers/queryRunner.ts

```diff
@@ -91,6 +93,13 @@ export default class QueryRunner {
             this._vscodeWrapper = new VscodeWrapper();
         }
 
+        // Setup Logger
+
+        let channel = this._vscodeWrapper.createOutputChannel(
```

Comment:
do we need a separate logging channel?  I'm wonder if it's best practice to create new output channels per feature area?

Reply:
We should be using vscodeWrapper.outputChannel, which will reuse the existing MSSQL one.  Also, ideally adding a logging prefix to make logs easier to filter through: `Logger.create(channel, "Querys")` or maybe "QueryRunner"